packages/sign_in_with_apple/sign_in_with_apple/pubspec.yaml

```diff
@@ -1,12 +1,12 @@
 name: sign_in_with_apple
 description: Flutter bridge to initiate Sign in with Apple (on iOS, macOS, and Android). Includes support for keychain entries as well as signing in with an Apple ID.
-version: 4.3.0
+version: 4.4.0
 homepage: https://github.com/aboutyou/dart_packages/tree/master/packages/sign_in_with_apple
 repository: https://github.com/aboutyou/dart_packages
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
-  flutter: ">=2.8.0"
+  sdk: ">=2.18.5 <3.0.0"
+  flutter: ">=3.7.6 <4.0.0"
```

Comment:
This would need to be released as v5 if we bump the flutter version to 3.x right?

Reply:
Yeah, had the same thought. I suppose that would make sense then with a clear breaking change dropping the legacy versions.

Even though it shouldn't cause any trouble (as client could not upgrade to this version), it's probably best to consciously make consumers aware of this upgrade anyway.

Reply:
Let's do v5 then :)